src/mb_enhanced_cover_art_uploads/seeding/atisket/dimensions.ts

```diff
@@ -0,0 +1,125 @@
+import pRetry from 'p-retry';
+
+import type { Dimensions, FileInfo } from '@src/mb_caa_dimensions/ImageInfo';
+import { LOGGER } from '@lib/logging/logger';
+import { safeParseJSON } from '@lib/util/json';
+import { gmxhr, HTTPResponseError } from '@lib/util/xhr';
+import { BaseImage } from '@src/mb_caa_dimensions/Image';
+
+// Use a multiple of 3, most a-tisket releases have 3 images.
+// Currently set to 30, should allow 10 releases open in parallel.
+export const MAX_CACHED_IMAGES = 30;
+
+export const CACHE_LOCALSTORAGE_KEY = 'ROpdebee_dimensions_cache';
+
+export interface CacheEntry {
+    dimensions?: Dimensions;
+    fileInfo?: FileInfo;
+    addedDatetime: number;
+}
+export type CacheStore = Record<string, CacheEntry>;
+
+// Simple local storage-backed info cache. We want to cache a image information
+// for a limited number of images to prevent reloading information that we
+// already loaded. E.g. we loaded image information on the main page, and now
+// the images are displayed again on the post-submit screen. However, we don't
+// need as large of a cache as CAA Dimensions, and we don't want to reuse the
+// IndexedDB implementation since it adds quite a few dependencies.
```

Comment:
Now that the Bandcamp provider is using the IndexedDB implementation that's no longer a valid argument 😉

Reply:
The retrying does add some dependencies, but we're using `p-retry` for a-tisket too (and it would probably make sense to sprinkle it into some other places too).